vllm/worker/cache_engine.py

```diff
@@ -34,7 +34,7 @@ def __init__(
         self.head_size = model_config.get_head_size()
         self.num_layers = model_config.get_num_layers(parallel_config)
         self.num_heads = model_config.get_num_kv_heads(parallel_config)
-        self.dtype = model_config.dtype
+        self.dtype = cache_config.cache_dtype if cache_config.cache_dtype else model_config.dtype
```

Comment:
Please change the following line and let the linter fix the line width.
```suggestion
        self.dtype = (cache_config.cache_dtype if cache_config.cache_dtype else model_config.dtype)
```

Reply:
Fixed

Reply:
> +1 Can we get this from model config?

It will be better to move this to model config. I will done in the next commit soon.